lab-davide/gulpfile.js

```diff
@@ -0,0 +1,13 @@
+var gulp = require('gulp');
+var mocha = require('gulp-mocha');
+
+
+gulp.task('default', function() {
+  return gulp.src('./test/greet-test.js', {read: false})
+  .pipe(mocha({reporter: 'nyan'}));
```

Comment:
What is written here is great, but you're missing gulp-eslint which will eslint your code  so a point is being deducted.